### ISSUE TITLE
Wiki [first version]

### ISSUE THREAD

**Issue by sgoia** [bepronetwork/bepro-js#27]
Have a wiki with everything explained should anyone need to understand certain aspects from the project or the dev environment setup and usage, tests etc.

**Comment by sgoia** [comment]
working on it.

**Comment by sgoia** [comment]
wiki is at this forked repo ready for merging https://github.com/sgoia/bepro-js.wiki.git

**Comment by teedavies** [comment]
https://github.com/sgoia/bepro-js/wiki

**Comment by teedavies** [comment]
I hope it is done

**Comment by sgoia** [comment]
yes, simple 1st version with things to keep in mind, more will be added as needed.
